transit-ui/src/components/hardcode_json.py

```diff
@@ -0,0 +1,61 @@
+import json
```

Comment:
Is this a test script to generate `dict.json`? Lets make a new folder under `transit-ui` called `test` and put this there instead.

Reply:
yes, this is a test script. i moved the output into a new folder named `data`.

Reply:
moved `hardcode_json.py` into `src/test`